Composer/packages/extensions/visual-designer/src/widgets/DialogRefCard.tsx

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import { generateSDKTitle } from '@bfc/shared';
+import get from 'lodash/get';
+
+import { FormCard } from '../components/nodes/templates/FormCard';
+import { WidgetContainerProps, WidgetComponent } from '../schema/uischema.types';
+import { ObiColors } from '../constants/ElementColors';
+import { NodeEventTypes } from '../constants/NodeEventTypes';
+import { NodeMenu } from '../components/menus/NodeMenu';
+
+export interface DialogRefCardProps extends WidgetContainerProps {
+  dialog: string | object;
+  getRefContent?: (dialogRef: JSX.Element) => JSX.Element;
+  colors?: {
+    theme: string;
+    icon: string;
+  };
+}
+
+const DefaultCardColor = {
+  theme: ObiColors.AzureGray3,
+  icon: ObiColors.AzureGray2,
+};
+
+export const DialogRefCard: WidgetComponent<DialogRefCardProps> = ({
+  id,
+  data,
+  onEvent,
+  dialog,
+  getRefContent,
+  colors = DefaultCardColor,
+}) => {
+  const header = generateSDKTitle(data);
+  const nodeColors = { themeColor: colors.theme, iconColor: colors.icon };
+  const calleeDialog = typeof dialog === 'object' ? get(dialog, '$ref') : dialog;
+  const dialogRef = (
+    <span
+      css={{
+        cursor: 'pointer',
+        color: 'blue',
```

Comment:
Could this use a Fabric variable instead? What about using the `Link` component?

Reply:
yes!

Reply:
![image](https://user-images.githubusercontent.com/8528761/72204979-c0b5fb00-34b8-11ea-8d58-3e0af3d67f79.png)

Reply:
Thanks. This gets us easy A11y support.